website/client-old/js/services/guideServices.js

```diff
@@ -82,11 +82,11 @@ function($rootScope, User, $timeout, $state, Analytics) {
         }, {
           element: ".meter.mana",
           title: window.env.t('spells'),
-          content: window.env.t('spellsText') + " <a target='_blank' href='http://habitica.wikia.com/wiki/Todos'>" + window.env.t('toDo') + "</a>."
+          content: window.env.t('spellsText')
         }, {
           orphan: true,
           title: window.env.t('readMore'),
-          content: window.env.t('moreClass') + " <a href='http://habitica.wikia.com/wiki/Class_System' target='_blank'>Wikia</a>.",
+          content: window.env.t('moreClass'),
           final: true
```

Comment:
All of this is looking good and is ready to be merged except that I haven't tested the two changes just above here because I can't work out how to trigger that code. Is it actually in use anywhere?

FYI @crookedneighbor